arduino/smartcar/smartcar.ino

```diff
@@ -84,13 +84,11 @@ void setup() {
 }
  
 void takeInput(String input) {
-        int inputSelection = input.substring(0).toInt();
+        int inputSelection = input.toInt();
```

Comment:
This is still wrong and **coincidentally** works! 
If input is a string with spaces and what-not you shouldn't rely on `toInt` parsing the "first part" right. Get the part of the string that you want. If you want a particular character use [charAt](https://www.arduino.cc/reference/en/language/variables/data-types/string/functions/charat/)

Reply:
Of course, the option to use `substring` correctly is always there.

Reply:
Okay.. So we are utilizing subString correctly now I believe..